.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,69 @@
+name: ✨ Feature Request
+description: Suggest a new feature
+title: "[Feature]: <title>"
+labels: ["enhancement", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request!
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the requested feature.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: dropdown
+    id: user-personas
+    attributes:
+      label: User Personas affected by this feature
+      description: List of user peronas available [here](https://enarx.dev/docs/Technical/Cases)
+      multiple: true
+      options:
+        - Terry - "Elastic compute"
+        - Michelle - "Unused cycles"
+        - Bob - "National security"
+        - Kohani - "Key management"
+        - Filippe - "Data analytics"
+        - Charley - "Edge computing"
+        - Pierre - "Network security functions"
+        - Other
+    validations:
+      required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A concise description of the requested feature.
+    validations:
+      required: true
+  - type: textarea
+    id: reason
+    attributes:
+      label: Reason
+      description: Explain why we should add this feature.
+    validations:
+      required: true
```

Comment:
I would prefer to merge these into one section, again to foolproof this and to avoid getting issues like:
```
Description: awesome `$feature` should be supported by Enarx
Reason: ...
```

But I'm OK with having an optional reason.